### PR TITLE
map_object_graphics_info, map_object_graphics_info_pointers, map_object_pic_table

### PR DESCRIPTION
**Pull request by TaoNeutrino**
I did not remove the .inc files from which these come from, which are named the same except 'map' is 'field'

These are needed for pretmap

### CODE REVIEW

**Comment by Diegoisawesome** [comment]
Why not remove the original files? Any specific reason?

**Comment by TaoNeutrino** [comment]
I not sure if they are being used by any other files, I just wanted to get these up for pretmap

File: src/data/field_map_obj/map_object_graphics_info_pointers.h [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+    &gFieldObjectGraphicsInfo_Pikachu,
+    &gFieldObjectGraphicsInfo_Azumarill,
+    &gFieldObjectGraphicsInfo_Wingull,
+    &gFieldObjectGraphicsInfo_Kecleon2,
+    &gFieldObjectGraphicsInfo_TuberMSwimming,
+    &gFieldObjectGraphicsInfo_Azurill,
+    &gFieldObjectGraphicsInfo_Mom,
+    &gFieldObjectGraphicsInfo_LinkBrendan,
+    &gFieldObjectGraphicsInfo_LinkMay,
+	&gFieldObjectGraphicsInfo_Juan,
```
Comment (Diegoisawesome) [inline src/data/field_map_obj/map_object_graphics_info_pointers.h]: Tabs->spaces

**Comment by TaoNeutrino** [comment]
Finished two more

**Comment by huderlem** [comment]
This PR is now obsolete after #263

**Pull request closed without merging**